Plugins/Detray/include/Acts/Plugins/Detray/DetrayConversionHelper.hpp

```diff
@@ -95,19 +95,27 @@ inline static std::size_t accelerationLink(const binning_values_t& casts) {
   // Default is `brute_force`
   std::size_t accLink = detray::io::accel_id::brute_force;
   if (casts.size() == 2u) {
-    if (casts[0u] == binX && casts[1u] == binY) {
+    if (casts[0u] == Acts::BinningValue::binX &&
```

Comment:
FYI in C++20 you can do `using enum Acts::BinningValue` to bring the enum cases into a scope. Could be very handy here.